cpp/oneapi/dal/algo/pca/common.hpp

```diff
@@ -57,6 +58,39 @@ using v1::by_default;
 
 } // namespace method
 
+/// Represents result option flag
+/// Behaves like a regular :expr`enum`.
```

Comment:
Should it be :expr:`enum`?

Reply:
I think yes, since we have the same description in all algorithms, where there are optional results